loc/translations-import/vscode-powerplatform.zh-CN.xlf

```diff
@@ -48,22 +48,17 @@
       </trans-unit>
       <trans-unit id="pacCLI.authPanel.welcome.whenInteractiveSupported">
         <source xml:lang="en">No auth profiles found on this computer.
-[Add Admin Auth Profile](command:pacCLI.authPanel.newAdminAuthProfile)
 [Add Dataverse Auth Profile](command:pacCLI.authPanel.newDataverseAuthProfile)</source>
         <note>This is a Markdown formatted string, and the formatting must persist across translations.
-The second line should be '[TRANSLATION HERE](command:pacCLI.authPanel.newAdminAuthProfile)', keeping brackets and the text in the parentheses unmodified
-The third line should be '[TRANSLATION HERE](command:pacCLI.authPanel.newDataverseAuthProfile)', keeping brackets and the text in the parentheses unmodified</note>
-        <target state="translated">在此计算机上找不到身份验证配置文件。
-[添加管理员身份验证配置文件](command:pacCLI.authPanel.newAdminAuthProfile)
-[添加 Dataverse 身份验证配置文件](command:pacCLI.authPanel.newDataverseAuthProfile)</target>
+The second line should be '[TRANSLATION HERE](command:pacCLI.authPanel.newDataverseAuthProfile)', keeping brackets and the text in the parentheses unmodified</note>
       </trans-unit>
       <trans-unit id="pacCLI.authPanel.welcome.whenInteractiveNotSupported">
```

Comment:
Missing translations for `pacCLI.authPanel.welcome.whenInteractiveNotSupported` across all languages.

Reply:
Ignoring for now, Translation team will get the string in another PR